frc1388-2024/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -42,13 +77,19 @@ public RobotContainer() {
    * joysticks}.
    */
   private void configureBindings() {
-    // Schedule `ExampleCommand` when `exampleCondition` changes to `true`
-    new Trigger(m_exampleSubsystem::exampleCondition)
-        .onTrue(new ExampleCommand(m_exampleSubsystem));
+    DriveCommand m_driveCommand = new DriveCommand(
+      m_driveTrain, 
+      () -> m_driverController.getLeftY(), 
+      () -> m_driverController.getLeftX(), 
+      () -> m_driverController.getRightX()
+    );
+
+    m_driveTrain.setDefaultCommand(m_driveCommand);
+
+    
+    m_driverController.a().onTrue(new InstantCommand(() -> m_driveTrain.resetGyroHeading()));
+    m_driverController.a().onTrue(new InstantCommand(() -> m_driveTrain.resetPose(new Pose2d())));
```

Comment:
Unsure if both of these button bindings work the way intended, both buttons work at the same time.